Unit3FinalAssessment/Unit3FinalAssessment/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,123 @@
+                                        <constraint firstItem="WiK-7K-VbF" firstAttribute="leading" secondItem="4WR-Dt-GYI" secondAttribute="leading" constant="8" id="soz-uZ-HtM"/>
+                                        <constraint firstItem="UwB-yR-gEj" firstAttribute="top" secondItem="hYS-Fo-DRM" secondAttribute="bottom" constant="2" id="y7M-xi-J1A"/>
+                                    </constraints>
+                                </tableViewCellContentView>
+                                <connections>
+                                    <outlet property="objectImageView" destination="WiK-7K-VbF" id="JLK-dh-Sc7"/>
+                                    <outlet property="objectSubtitleLabel" destination="UwB-yR-gEj" id="wTp-Qe-eG6"/>
+                                    <outlet property="objectTitleLabel" destination="hYS-Fo-DRM" id="QGm-B1-hN5"/>
+                                    <segue destination="zMS-fE-eaw" kind="show" id="Pbq-OB-qeL"/>
+                                </connections>
+                            </tableViewCell>
+                        </prototypes>
+                        <connections>
+                            <outlet property="dataSource" destination="6oN-3b-hRz" id="M49-XW-4G2"/>
+                            <outlet property="delegate" destination="6oN-3b-hRz" id="6Do-1y-Tv0"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" title="Rings without the Lords" id="I8J-8b-o51"/>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="FjP-ZW-GEK" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="764" y="129.98500749625188"/>
+        </scene>
+        <!--Detail View Controller-->
+        <scene sceneID="QCc-48-Hik">
+            <objects>
+                <viewController id="zMS-fE-eaw" customClass="DetailViewController" customModule="Unit3FinalAssessment" customModuleProvider="target" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="3lQ-Ja-4fw"/>
+                        <viewControllerLayoutGuide type="bottom" id="L4V-Ck-tNo"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="1Fe-De-yk2">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="Bn8-83-z8e">
```

Comment:
This image content mode should be aspect fit or aspect fill. Scale to fill distorts.